pkg/controller/kubelet-config/kubelet_config_features.go

```diff
@@ -182,48 +168,41 @@ func (ctrl *Controller) deleteFeature(obj interface{}) {
 // generateFeatureMap returns a map of enabled/disabled feature gate selection with exclusion list
 //
 //nolint:gocritic
-func generateFeatureMap(features *osev1.FeatureGate, exclusions ...string) (*map[string]bool, error) {
+func generateFeatureMap(featuregateAccess featuregates.FeatureGateAccess, exclusions ...osev1.FeatureGateName) (*map[string]bool, error) {
 	rv := make(map[string]bool)
```

Comment:
Should probably update this at some point to use the new `configv1.FeatureGateName` instead of a string